image-pipeline/expression-scatter-plots.js

```diff
@@ -392,26 +430,24 @@ async function run() {
   const accession = values.accession
   print(`Accession: ${accession}`)
 
-  startTime = Date.now()
+  await uploadLog()
 
   const crum = 'isImagePipeline=true'
   // Get list of all genes in study
-  const exploreApiUrl = `${origin}/single_cell/api/v1/studies/${accession}/explore?${crum}`
+  const exploreApiUrl = `${fetchOrigin}/single_cell/api/v1/studies/${accession}/explore?${crum}`
   print(`Fetching ${exploreApiUrl}`)
 
   const response = await fetch(exploreApiUrl)
 
-  // TODO (SCP-4666): Use plain-old response.json() without all this log noise
-  // once we can access staging SCP API from staging-project PAPI VM.
-  print('response.status')
-  print(response.status)
-  const text = await response.text()
-  print('response text')
-  print(text)
+  // Helpful for debugging errors
+  // const text = await response.text()
+  // print('response text')
+  // print(text)
 
   let json
   try {
-    json = JSON.parse(text)
+    // json = JSON.parse(text)
```

Comment:
Does this comment need to remain?

Reply:
Yes, it helps.  I just added a comment to indicate it's not a straggler in f6e1698.